src/main/webapp/i18n/de/course.json

```
@@ -41,8 +41,8 @@
                 "description": "Diese Nummer gibt an, wie viele Zeichen für die Antwort pro Beschwerde maximal verwendet werden darf."
             },
             "postsEnabled": {
-                "title": "Posts von Studierenden aktivieren",
-                "description": "Postings auf Vorlesung- und Übungsseite aktivieren / deaktivieren"
+                "title": "Kommunikationsfunktionen aktivieren",
+                "description": "Kommunikationsbereich auf Vorlesungs- und Übungsseiten aktivieren."
```

```suggestion
                "description": "Kommunikationsbereich für Vorlesungs- und Übungsseiten aktivieren."
```
I think that sounds better imo :thinking: